fvtest/porttest/omrstrTest.cpp

```diff
@@ -1538,35 +1546,71 @@ TEST(PortStrTest, str_test_atoe_vsnprintf)
 
 	/* Neither min_width nor precision is specified for a null terminated input string*/
 	portTestEnv->log("\n\tTesting case 1\n");
-	rc |= printToBuffer(OMRPORTLIB, TRUE, buffer0, bufferLength0, expectedOutput1, "%s", nullTerminatedString);
+	rc |= printToBuffer(portLibrary, TRUE, buffer0, bufferLength0, expectedOutput1, strlen(expectedOutput1), "%s", nullTerminatedString);
 
 	/* min_width is less than the length of inputString */
 	portTestEnv->log("\n\tTesting case 2\n");
-	rc |= printToBuffer(OMRPORTLIB, FALSE, buffer2, bufferLength2, expectedOutput2, "%*s", 2, nonNullTerminatedString);
+	rc |= printToBuffer(portLibrary, FALSE, buffer2, bufferLength2, expectedOutput2, strlen(nullTerminatedString), "%*s", 2, nullTerminatedString);
```

Comment:
if i use non terminated string, how the method would know how many chars it should take from the argument? the is no termination to stop it and no `precision` is provided to limit it. It would cause segfault. It was passing before because the provided `count` was stoping it which was not correct and has changed in this pr